src/report_reader.cpp

```diff
@@ -369,24 +397,12 @@ DataFrame<T> ReportReader<T>::Population::get(const nonstd::optional<Selection>&
             .read(data);
         int timer_index = 0;
         for (const std::vector<float>& datum : data) {
-            for (float d : datum) {
-                data_frame.data[timer_index].push_back(d);
-            }
+            std::copy(datum.data(),
+                      datum.data() + datum.size(),
+                      &data_frame.data[timer_index * data_frame.n_rows + ids_index]);
```

Comment:
I believe there was a bug here. The flattened structure changes row by doing `index*n_cols`, not `index*n_rows`

Reply:
Actually there is a general naming swap rows-columns... Improving this